src/groupeddataframe/grouping.jl

```diff
@@ -864,9 +864,14 @@ check_aggregate(::typeof(minimum)) = Reduce(min)
 check_aggregate(::typeof(mean)) = Reduce(Base.add_sum, nothing, /)
 check_aggregate(::typeof(sum∘skipmissing)) = Reduce(Base.add_sum, !ismissing)
 check_aggregate(::typeof(prod∘skipmissing)) = Reduce(Base.mul_prod, !ismissing)
-check_aggregate(::typeof(maximum∘skipmissing)) = Reduce(max, !ismissing)
-check_aggregate(::typeof(minimum∘skipmissing)) = Reduce(min, !ismissing)
 check_aggregate(::typeof(mean∘skipmissing)) = Reduce(Base.add_sum, !ismissing, /)
+# Empty groups can only happen with skipmissing and are only a problem for min and max
+function check_notempty(res, count::Integer)
```

Comment:
This approach has the side effect of making an unnecessary copy of the result. Maybe I can find something better...

Reply:
I've pushed a new commit which avoids the copy.